code/__HELPERS/logging.dm

```diff
@@ -36,6 +36,22 @@
 	GLOB.STUI.admin.Add("\[[time_stamp()]]ADMIN: [text]")
 	GLOB.STUI.processing |= STUI_LOG_ADMIN
 
+/proc/check_slur(var/text, var/mob/source, var/method = "Speech")
+	if(!CONFIG_GET(flag/log_slurs))
+		return FALSE
+	var/list/TriggerWords = file2list("config/banned_words.txt")
+	for(var/Trigger in TriggerWords)
+		if(findtext(text,Trigger))
```

Comment:
gotta make sure this won't trigger false positives for things like the word "sniggers"

Reply:
please use regex for this instead

Reply:
If you need an example of how to do this, please check https://github.com/tgstation/tgstation/blob/6ce946a0566cb6c19bffb588c331107ef54e04ee/code/controllers/configuration/configuration.dm#L448